ibm/mas_devops/playbooks/cp4d.yml

```diff
@@ -11,6 +11,7 @@
     install_watson_openscale: "{{ lookup('env', 'CPD_INSTALL_OPENSCALE') | default('False', true) | bool }}"
     install_watson_discovery: "{{ lookup('env', 'CPD_INSTALL_DISCOVERY') | default('False', true) | bool }}"
     install_spss_modeler: "{{ lookup('env', 'CPD_INSTALL_SPSS') | default('False', true) | bool }}"
+    install_cognos_analytics: "{{ lookup('env', 'CPD_INSTALL_COGNOS') | default('False', true) | bool }}"
     cpd_product_version: "{{ lookup('env', 'CPD_PRODUCT_VERSION') | default('4.5.0', true) }}"
```

Comment:
While you are making updates here, can you change the default version to the latest please?